compal/__init__.py

```diff
@@ -601,6 +768,34 @@ def set_remoteaccess(self, enabled, port=8443):
             ('Port', port)
         ]))
 
+    def set_forgot_pw_email(self, email_addr):
+        """
+        Set email address for Forgot Password function
+        """
+        return self.modem.xml_setter(Set.SET_EMAIL, OrderedDict([
+            ('email', email_addr),
+            ('emailLen', len(email_addr)),
+            ('opt', 0)
+        ]))
+
+    def send_forgot_pw_email(self, email_addr):
+        """
+        Send an email to receive new or forgotten password
+        """
+        return self.modem.xml_setter(Set.SEND_EMAIL, OrderedDict([
+            ('email', email_addr),
+            ('emailLen', len(email_addr)),
```

Comment:
That is a really suspicious parameter, given that the UI is a badly written C program